style.css

```diff
@@ -102,3 +137,246 @@ span {
   color: #505f79;
   width: 1.2rem;
 }
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  gap: 2rem;
+  margin-left: 1rem;
+}
+
+.card1,
+.card2,
+.card3,
+.card4 {
+  display: flex;
+  flex-direction: column;
+  border: 0.063rem solid #dfe1e6;
+  border-radius: 1rem;
+  background: #fff;
+  height: auto;
+  padding: 1rem;
+  width: 84%;
+}
+
+.mobileimage1 {
+  background-image: url('/images/mobileImg1.jpg');
+  background-repeat: no-repeat;
+  height: 13.75rem;
+  width: 100%;
+  border-radius: 0.5rem;
+}
+
+.mobileimage2 {
+  background-image: url('/images/mobileImg2.jpg');
+  background-repeat: no-repeat;
+  height: 13.75rem;
+  width: 100%;
+  border-radius: 0.5rem;
+}
+
+.mobileimage3 {
+  background-image: url('/images/mobileImg3.jpg');
+  background-repeat: no-repeat;
+  height: 13.75rem;
+  width: 100%;
+  border-radius: 0.5rem;
+}
+
+.mobileimage4 {
+  background-image: url('/images/mobileImg4.jpg');
+  background-repeat: no-repeat;
+  height: 13.75rem;
+  width: 100%;
+  border-radius: 0.5rem;
+}
+
+.mobile {
+  display: flex;
+  width: 100%;
+  gap: 1.5rem;
+  line-height: 0.5rem;
+  margin-top: -0.5rem;
+}
+
+.mobile li {
+  list-style: disc;
+  font-size: 0.813rem;
+  font-style: normal;
+  font-weight: 600;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  color: #7a869a;
+}
+
+.mobile .company {
+  list-style-type: none;
+  font-size: 0.813rem;
+  color: #344563;
+  font-style: normal;
+  font-weight: 600;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin-left: -2.25rem;
+}
+
+.mobile-paragraph {
+  color: #344563;
+  margin-left: -1.25rem;
+  line-height: 1.5rem;
+  font-size: 0.938rem;
+  font-style: normal;
+  font-weight: 400;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  width: auto;
+}
+
+.teck {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-left: -0.5rem;
+}
+
+.teck-list {
+  display: flex;
+  gap: 0.5rem;
+  list-style-type: none;
+  margin-left: -2rem;
+}
+
+.mobile-btn {
+  border: 0.1rem solid #ebebff;
+  border-radius: 0.5rem;
+  background-color: #e5e6ecec;
+  color: #6070ff;
+}
+
+.see-mobile-btn {
+  border: 0.5rem solid #ebebff;
+  border-radius: 0.5rem;
+  height: 2.5rem;
+  font-size: 1rem;
+  color: #6070ff;
+  margin-top: 0.25rem;
+  width: fit-content;
+}
```

Comment:
For good user experience, kindly add hover, enabled and pressed states to all the buttons in your project. Below is a screenshot from figma. 👍 

<img width="375" alt="Untitled" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/103386183/82e9208c-9ccd-4924-9ea0-9662b935934e">